test/fixtures/v5/merge/left_exit.json

```diff
@@ -0,0 +1,22 @@
+{
+    "step": {
+        "maneuver": {
+            "type": "merge",
+            "modifier": "left"
+        },
+        "name": "Way Name",
+        "exits": "4A,4B"
```

Comment:
I don't think a `merge` and `exit` should ever be able to be in the same instruction.